standard/schema/codelists/partyRole.csv

```diff
@@ -1,11 +1,11 @@
 Code,Title,Description
 buyer,Buyer,"A buyer is an entity whose budget will be used to pay for goods, works or services related to a contract."
-procuringEntity,Procuring Entity,"The entity managing the procurement. This may be different from the buyer who pays for, or uses, the items being procured."
+procuringEntity,Procuring entity,"The entity managing the procurement. This may be different from the buyer who pays for, or uses, the items being procured."
 supplier,Supplier,"An entity awarded or contracted to provide goods, works or services."
-tenderer,Tenderer,All entities who submit a tender
+tenderer,Tenderer,All entities who submit a tender.
 funder,Funder,The funder is an entity providing money or finance for this contracting process.
 enquirer,Enquirer,A party who has made an enquiry during the enquiry phase of a contracting process.
-payer,Payer,A party making a payment from a transaction
-payee,Payee,A party in receipt of a payment from a transaction
-reviewBody,Review Body,A party responsible for the review of this procurement process. This party often has a role in any challenges made to the contract award.
+payer,Payer,A party making a payment from a transaction.
+payee,Payee,A party in receipt of a payment from a transaction.
+reviewBody,Review body,A party responsible for the review of this procurement process. This party often has a role in any challenges made to the contract award.
 interestedParty,Interested Party,"A party that has expressed an interest in the contracting process: for example, by purchasing tender documents or submitting clarification questions."
```

Comment:
'Interested Party' -> 'Interested party'

Reply:
Fixed in fa2be67